src/Entity/JobOffer.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\JobOfferRepository")
+ */
+final class JobOffer
+{
+    const STATUS_PENDING_REVIEW = 'PENDING_REVIEW';
+    const STATUS_APPROVED = 'APPROVED';
+
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=50)
+     */
+    private $title;
+
+    /**
+     * @ORM\Column(type="text")
+     */
+    private $description;
+
+    /**
+     * @ORM\Column(type="string", length=50)
+     */
+    private $company;
+
+    /**
+     * @ORM\Column(type="string", length=10)
+     */
+    private $seniorityLevel;
+
+    /**
+     * @ORM\Column(type="integer")
+     */
+    private $salary;
```

Comment:
Money type? May float?

Reply:
Yes, we can improve that. I just started with bare minimum, we start with other cards. We can create a card for that.

Will we use a single value or a range like:

* 3500.49
Or:
* 2500.99 - 3000.05